### PR TITLE
Monthly electricity metric

### PR DESCRIPTION
**Pull request by gwenchee**
In this PR, I create a metric for monthly electricity generated.

### CODE REVIEW

**Comment by gwenchee** [comment]
I can't assign reviewers in the PR. So I'll ping y'all here. @katyhuff @bam241 @gtw2

File: tests/test_metrics.py [hunk 1]
```diff
@@ -322,6 +322,31 @@ def test_annual_electricity_generated_by_agent():
...
+        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 1, 3, 1200),
+        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 2, 3, 2400),
+        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 1, 12, 1200),
+        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 2, 12, 2400),
+        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 3, 12, 4800),
+        ], dtype=ensure_dt_bytes([
+                ('SimId', 'O'), ('AgentId', '<i8'), ('Time', '<i8'),
+                ('Value', '<f8')]))
+        )
+    obs = metrics.annual_electricity_generated_by_agent.func(tsp)
```
Comment (bam241) [inline tests/test_metrics.py]: I think you forgot to update some part.

I think you only have copy pasted the `test_annual_electricity_generated_by_agent()`

**Comment by gwenchee** [comment]
@bam241 Good catch! I accidentally overlooked those lines. It's fixed now.

**Comment by gwenchee** [comment]
merge it? @bam241

**Comment by bam241** [comment]
I was hoping some else could review/merge it ? cc @gonuke @katyhuff

**Comment by gonuke** [comment]
Thanks @gwenchee

**Pull request merged**